### PR TITLE
Fixes for Nexogen.Libraries.Metrics.Grpc 

### PR DESCRIPTION
**Pull request by bastianeicher**
This PR fixes the logic used to split gRPC service and method names.

I also adjusted the namespace to match the package name change from #46.

### CODE REVIEW

File: Nexogen.Libraries.Metrics.Grpc/MetricsBase.cs [hunk 1]
```diff
@@ -90,7 +90,10 @@ public void StreamMsgSent(MethodType type, string method)
 
         private static string[] GetLabels(MethodType type, string method)
         {
-            var split = method.Split('/');
+            var split = method.Split('.');
+            string serviceName = string.Join(".", split.Take(split.Length - 1));
```
Comment (ahoka) [inline Nexogen.Libraries.Metrics.Grpc/MetricsBase.cs]: Would you please add a unit test for this logic, with real world examples? I think that would be a valuable.

**Review by ahoka — CHANGES_REQUESTED** [review]
Thanks, I only have one comment. Sorry for missing the namespaces earlier.

**Comment by bastianeicher** [comment]
I've done some extensive refactoring to make the library more testable and added unit tests.

**Review by ahoka — APPROVED** [review]
Thanks.

**Comment by ahoka** [comment]
I have created a new release with these changes and unlisted the old version with the incorrect namespace.

**Comment by bastianeicher** [comment]
Thanks :)

**Pull request merged**